docs/configuration/tls.md

```diff
@@ -37,13 +39,19 @@ stringData:
   ca.crt: <contents-of-example.crt>
 ```
 
-For basic TLS, only the fields `tls.crt` and `tls.key` are needed in the kube secret. For mutual TLS, add `ca.crt` in the kube secret and set the configuration `tls.clientAuth` to `require` in the ConfigMap `model-serving-config`.
+For basic TLS, only the fields `tls.crt` and `tls.key` are required. For mutual TLS, `ca.crt` should be included and `tls.clientAuth` should be set to `require` in the [`model-serving-config` ConfigMap](./README.md).
+
+You can also create this secret imperatively using:
+
+```
+kubectl create secret tls <secret-name> --cert <cert-file> --key <key-file>
```

Comment:
similar to above comment, let's make this an executable scriptlet, based on the vars and files defined earlier

```
kubectl create secret tls ${SECRET_NAME} --cert "example.crt" --key "example.key"
```

possibly rename the "example.*" files to something meaningful, like "server.crt" and "server.key" or similar.